packages/lumberaxe/lib/lumberaxe/log_chooser.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "active_support/tagged_logging"
+
+module Lumberaxe
+  class LogChooser
+    cattr_accessor :log_level
+
+    def self.logger(progname:)
+      logger = Logger.new(primary_logdev, progname: progname)
+      logger.level = log_level
+      logger.formatter = JSONFormatter.new if structured_logging?
+      logger
+    end
+
+    # Standard practice for applications running in Docker containers
+    # is to send their logging output to STDOUT instead of various
+    # logfiles on disk.
+    def self.primary_logdev
+      $stdout
+    end
```

Comment:
```suggestion
    # Standard practice for applications running in Docker containers
    # is to send their logging output to STDOUT instead of various
    # logfiles on disk.
```suggestion
    # Standard practice for applications running in Docker containers
    # is to send their logging output to STDOUT instead of various
    # logfiles on disk.
    cattr_accessor :primary_logdev do
      $stdout
    end
```

Just to be open for customization